### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by rubydevi**
**Description:**

In this project, I have continued with the development of the Math Magicians app. I have developed a React component that will hold the core functionality: a calculator.

**Changes Made:**

- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called `components`.
- Inside components, created a new `Calculator.js` file.
- In `Calculator.js`, created a React component that matches the provided design.
- Used `CSS` and `JSX` to design the `UI`
- Used functional components
- Created at least 1 child component

Please review my code, your feedback is highly appreciated.

### CODE REVIEW

File: src/components/Calculator.js
Comment (wayungi) [inline src/components/Calculator.js:1]: - [ ] It is a project requirement that Calculator component should have at least 1 child component. I suggest you create the button as a component and import & use it here 👍🏾

Comment (wayungi) [reply src/components/Calculator.js:1]: -  [OPTIONAL] I suggest that you give the calculator a definate size according to the project design. A fluid calculator witll be hard to style if the page has other components.  👍🏾 
calc

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @rubydevi , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡  
- [ ] Good commit messages ✔
- [ ]  Correct Git flow ✔️ 
- [ ] Descriptive PR body ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly delete the following filesbecause they are considered **boiler plate code** `App.test.js, index.css, setupTest.js, ...` & all others that are auto generated but not used in this PR 😺  
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
@rubydevi, there seems to be an error on starting your application.

https://www.loom.com/share/59ec9d02bf0c4f2580f735ca4346f19a

**Review by topeogunleye — APPROVED** [review]
### Hi @rubydevi, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- The calculator component has a child component. ✔️ 
- App runs ✔️ 
image

 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**